extensions/binary_glTF/README.md

```diff
@@ -1,10 +1,12 @@
-# CESIUM_binary_glTF
+# binary_glTF
```

Comment:
We'll still want to leave `CESIUM_binary_glTF` around at least for a bit since we have _lots_ of Cesium users depending on it.  Our goal is to move completely to `binary_glTF` and then not try to ratify or promote `CESIUM_binary_glTF`.